### ISSUE TITLE
[webgl] CPU forwarding should work for unmodularized kernels.

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#3994]
**Describe the current behavior**
On master, CPU forwarding will fail in WebGL for ops that are unmodularized in WebGL but modularized in CPU.

**Describe the expected behavior**
CPU forwarding should work.

**Standalone code to reproduce the issue**
```
const x = tf.softmax([1, 2, 3, 4]);
```
The above code fails on the WebGL backend.

**Comment by jinjingforever** [comment]
Hi Ann,

Quick update: I identified several ops that need to be fixed (e.g. ceil, floor, exp, etc). Once those are fixed, the ops that use them (e.g. softmax) should also be fixed too. I also saw that your PR #3956 fixed a couple of ops as well:) Thanks!

To reduce conflicts, I will wait for your PR to be submitted before sending out my fix. Please take your time!

Thanks Ann!

**Comment by annxingyuan** [comment]
Sounds good @jinjingforever - thank you for taking this on!
